File: src/com/tal/android/plugin/github/ui/tablemodels/GHRepoTableModel.java
package com.tal.android.plugin.github.ui.tablemodels;

import java.io.Serializable;
import java.util.List;

import com.tal.android.plugin.github.domain.gh.GHRepositoryWrapper;

public class GHRepoTableModel extends BaseModel<GHRepositoryWrapper> implements Serializable {

    public GHRepoTableModel(List<GHRepositoryWrapper> repositories, String[] colNames) {
        super(repositories, colNames);
    }

    @Override
    public Class<?> getColumnClass(int columnIndex) {
        return GHRepositoryWrapper.class;
    }
}

File: src/com/nerdscorner/android/plugin/github/ui/tablemodels/GHRepoTableModel.kt
package com.nerdscorner.android.plugin.github.ui.tablemodels

import com.nerdscorner.android.plugin.github.domain.gh.GHRepositoryWrapper
import java.io.Serializable

class GHRepoTableModel(repositories: MutableList<GHRepositoryWrapper>, colNames: Array<String>) :
        BaseModel<GHRepositoryWrapper>(repositories, colNames), Serializable {

    override fun getColumnClass(columnIndex: Int): Class<*> {
        return GHRepositoryWrapper::class.java
    }

    override fun compare(one: GHRepositoryWrapper, other: GHRepositoryWrapper) = super.compare(other, one)
}
